server/src/main/java/com/lunchtime/controllers/FeedbackController.java

```diff
@@ -0,0 +1,29 @@
+package com.lunchtime.controllers;
+
+import java.util.List;
+import com.lunchtime.models.Feedback;
+import com.lunchtime.service.FeedbackService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+
+@RestController
+@RequestMapping("/api/restaurants")
```

Comment:
There is RestaurantResource class exists, it has endpoint `restaurants`. You don't need to create another controller class with the same endpoint. You can just update first one

Reply:
we need two controller one restaurants, one feedback. I can delete this @RequestMapping("/api/restaurants")

Reply:
Yes, we need controllers for Restaurant, Feedback, Event, so on